src/openforms/forms/models/form.py

```diff
@@ -444,3 +446,37 @@ class FormLogic(models.Model):
         ),
         default=False,
     )
+
+
+class FormsExport(DeleteFileFieldFilesMixin, models.Model):
+    export_content = PrivateMediaFileField(
+        verbose_name=_("export content"),
+        upload_to="exports/%Y/%m/%d",
+        help_text=_("Zip file containing all the exported forms"),
+    )
+    downloaded = models.BooleanField(
```

Comment:
this can be implicitly derived as `self.date_downloaded is not None`, no need for this field I believe.